lua/efmls-configs/linters/markdownlint.lua

```diff
@@ -0,0 +1,13 @@
+local fs = require("efmls-configs.fs")
+
+local linter = "markdownlint"
+local command = string.format("%s --config %q --stdin", fs.executable(linter))
```

Comment:
missing an arg for `%q`? also why not let markdownlint handle looking up the config file on its own

Reply:
sorry, that one was a leftover from my personal config, where I added a config file. fixed.